packages/admin/src/pages/index.page.tsx

```diff
@@ -14,14 +16,8 @@ export const getServerSideProps: GetServerSideProps = async ({
   let response;
   try {
     response = await authenticateUser(cookieValue);
-  } catch (error) {
-    console.error('Failed to verify token', error);
-    return {
-      redirect: {
-        destination: getLoginUrl(),
-        permanent: false,
-      },
-    };
+  } catch (error: unknown) {
```

Comment:
this be error: AxiosError too or else we will get a type error down below

Reply:
You cannot actually add types to arguments in catch
>Catch clause variable type annotation must be 'any' or 'unknown' if specified.ts(1196)